src/zarpaint/napari.yaml

```
@@ -42,3 +45,5 @@ contributions:
     display_name: Split With Watershed
   - command: zarpaint.copy_data
     display_name: Copy Data
+  - command: zarpaint.interpolate_widg
+    display_name: Interpolate Slices
```

```suggestion
    display_name: Interpolate Slices with Signed Distance Transform
```

Just preparing for when we add future methods. I also hate that it's so hard for me to find out in other interpolation tools what method they're actually using!

---

I feel like when we have future methods, there should be a dropdown selection box in the interpolation widget to switch between them (instead of creating additional widgets, which this comment seems to suggest).